resources/views/relatorio/edit.blade.php

```diff
@@ -159,6 +263,105 @@
     @push ('scripts')
         <script>
             CKEDITOR.replace('relatorio');
+
+                $('#customFilearquivo').change(function(e){
+            var fileName = e.target.files[0].name;
+            $('.custom-file-label').html(fileName);
+            });
+            var $videoId = 1;
+            CKEDITOR.replace('denuncia-ckeditor', {
+            height: 400
+            });
+
+            function addImagem() {
+            if($('#imagens').children().length >= 20){
+                alert("A quantidade máxima de imagens é 20!");
+            }else{
+                var indice = document.getElementById("imagem_indice");
+                var imagem_indice = parseInt(document.getElementById("imagem_indice").value)+1;
+                indice.value = imagem_indice;
+
+                var campo_imagem = `<div class="col-md-4">
+                                        <div class="">
+                                            <label for="file-input-`+imagem_indice+`">
+                                                <img id="imagem_previa`+imagem_indice+`" class="img-fluid" src="{{asset('/img/nova_imagem.PNG')}}" alt="imagem de anexo" style="cursor: pointer;"/>
+                                            </label>
+                                            <input style="display: none;" type="file" name="imagem[]" id="file-input-`+imagem_indice+`" accept="image/*" onchange="loadPreview(event, `+imagem_indice+`)">
+                                        </div>
+                                        <div class="d-flex justify-content-end">
+                                            <div class="col-md-6" style="text-align: right">
+                                                <div id="nome`+imagem_indice+`" style="display: none; font-style: italic;"></div>
+                                            </div>
+                                            <div class="col-md-6" style="text-align: right">
+                                                <a style="cursor: pointer; color: #ec3b3b; font-weight: bold;" onclick="removerImagem(this, `+imagem_indice+`)">remover</a>
+                                            </div>
+                                        </div>
```

Comment:
se essa parte comentada não for usada, remover. Pois esse comentario padrão sempre dá problema com aplicações em laravel